lib/xayn_card_view/card_view.dart

```diff
@@ -262,7 +266,7 @@ class CardViewState extends State<CardView> {
           _isAbsorbingPointer = false;
 
           widget.controller?.index = _index;
-          print('$_index vs ${widget.itemCount}');
+          debugPrint('$_index vs ${widget.itemCount}');
```

Comment:
we can remove this, mistakenly left the print in there :)

Reply:
removed it 👍